lms/urls.py

```diff
@@ -1020,5 +1020,5 @@
 
 # Enhanced Staff Grader (ESG) URLs
 urlpatterns += [
-    url(r'^api/ora_staff_grader', include('lms.djangoapps.ora_staff_grader.urls', 'ora-staff-grader')),
+    url(r'^api/ora_staff_grader/', include('lms.djangoapps.ora_staff_grader.urls', 'ora-staff-grader')),
```

Comment:
good change with the slash, but again, we're moving toward using `path` over `url`